extension/src/prototypes/view/ui-containers/SelectionPanel.tsx

```diff
@@ -30,6 +32,13 @@ export const SelectionPanel: FC = () => {
 
   const findRootLayer = useSetAtom(findRootLayerAtom);
 
+  const legendComponentAtom =
+    (selectionGroups[0]?.type === COLOR_SCHEME_SELECTION ||
+      selectionGroups[0]?.type === IMAGE_SCHEME_SELECTION) &&
+    (selectionGroups[0]?.values[0]?.componentAtoms?.find(
+      c => c.type === "LEGEND_DESCRIPTION_FIELD",
+    ) as ComponentAtom<"LEGEND_DESCRIPTION_FIELD"> | undefined);
```

Comment:
This should be handled within LegendDescriptionSection. Also you can use [useFindComponent hook](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/9f1585ebc956b8566b4f871abd33b01c2ecc77bc/extension/src/shared/view-layers/hooks.ts#L7) like [here](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/9f1585ebc956b8566b4f871abd33b01c2ecc77bc/extension/src/shared/layerContainers/floodModel.tsx#L111).

Reply:
Updated!